spec/v3/queries/cron_spec.rb

```diff
@@ -20,6 +22,14 @@
       expect(query.start_all).to_not include(cron)
       expect(Travis::API::V3::Models::Cron.where(id: cron.id).length).to equal(0)
     end
+    
+    it 'enques error into a thread' do
+      error = StandardError.new('Konstantin broke all the thingz!')
+      Travis::API::V3::Models::Cron.any_instance.stubs(:branch).raises(error)
+      Raven.expects(:send_event).with do |event|
+        event.message == "#{error.class}: #{error.message}"
+      end
+      expect { get "/repo/#{repo.id}/crons" }.to raise_error(error)
```

Comment:
I think you want to call query.start_all here with no expectation as we are expecting Raven to be called with capture exception above.